src/components/molecules/CreateVenueModal/CreateVenueModal.scss

```diff
@@ -0,0 +1,81 @@
+@import "scss/constants.scss";
+
+.modal-backdrop {
+  opacity: 0.9 !important;
+  background-color: black !important;
+}
+
+.modal-content {
+  background-color: $saturated-black !important;
+}
```

Comment:
Why are we using `!important` here? That's not something we want to be doing.

Reply:
(Also the other places within this file where it's being used currently)

Reply:
The bootstrap modal is terrible when it comes to custom styles, and this seems to be the only way to override them. There are 2 props `dialogClassName` and `backdropClassName` which are okey, but they both have parent `classNames` which are the ones that I override with `!important`

I will take a look again because I'm not a fan of `!important` hopefully there is something else as alternative.

Reply:
Theres many other ways to get our styles to override. Checkout:

- https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
@0xdevalias I got rid of the second `className` and used the prop `backdropClassName` as an alternative for the first one. But it still needs `!important`. For some strange reason the default styles override the ones provided by the prop class name.

Reply:
Found another solution which seems to be working fine. It's `modal-content.in` and then the styles you want. No need for `!important` 🥳

Reply:
Nice!